core/systems/input/handheld_gamepad.gd

```diff
@@ -16,6 +16,8 @@ var queued_events: Array[InputDeviceEvent]
 ## List of ogui events that are currently held.
 var sent_ogui_events: PackedStringArray
 var _last_time := 0
+var _last_volume := 0
```

Comment:
Instead of tracking the volume state in the handheld gamepad, let's add methods to `AudioManager` to toggle mute, increase volume, etc.

Reply:
Completed. I had to make some additional changes as this exposed a flaw, it would mute on key down and unmute on key up.